src/assets/usage-examples-code/sprites/draw_sprite/draw_sprite-2-draw-sprite.txt

```diff
@@ -0,0 +1,3 @@
+### Draw_sprite() usage exaple
+
```

Comment:
Correct the spelling of "example"

Reply:
I have corrected the spelling. Thank You.